flink/flink-scala-2.12/pom.xml

```diff
@@ -578,6 +999,13 @@
       <artifactId>slf4j-api</artifactId>
     </dependency>
 
+    <dependency>
+      <groupId>com.lmax</groupId>
+      <artifactId>disruptor</artifactId>
```

Comment:
hive enabled log4j2 async log by default, which requires disruptor available on classpath

Reply:
A code comment would be useful here.

Reply:
added